models/biocro/R/write.configs.BIOCRO.R

```diff
@@ -46,18 +46,10 @@ convert.samples.BIOCRO <- function(trait.samples) {
   ## transform values with different units cuticular conductance - BETY default is
   ## umol; BioCro uses mol
   if ("b0" %in% trait.names) {
-    trait.samples <- transform(trait.samples, b0 = udunits2::ud.convert(b0, "umol", "mol"))
+    trait.samples$b0 = udunits2::ud.convert(trait.samples$b0, "umol", "mol")
   }
   if ("Sp" %in% trait.names) {
-    trait.samples <- transform(trait.samples, Sp = udunits2::ud.convert(Sp, "kg/m2", "g/cm2"))
-  }
-  if ("vmax" %in% trait.names) {
```

Comment:
FWIW the following HAAAKs don't don't do anything. They were originally introduced in https://github.com/infotroph/pecan/commit/ecee6fd45c889ce10151fd7ae8abda53c658c2b4#diff-fb09778b34d1b5614bf2c73c9b2b0591 to account for bugs in the biocro code by rescaling Rd and vmax. When the bug was resolved I removed the scaling in https://github.com/infotroph/pecan/commit/4b0340b1605198832107fdbde1ef27be860d5f2e  but should have just removed these lines alltogether.

Reply:
So OK to merge - thanks for catching this @infotroph!